src/System.Security.Cryptography.Xml/src/Resources/Strings.resx

```diff
@@ -1,5 +1,64 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
VS did this - I won't fight it since it will add it back each time we modify this time